source/includes/api/_widget.md

```diff
@@ -582,281 +595,543 @@ curl -X GET https://api.resourcewatch.org/v1/widget?includes=metadata
+- the user must match one of the following:
+  - have role `ADMIN`
+  - have role `MANAGER` and be the widget's owner (through the `userId` field of the widget)
+
+
+
+## Widget reference
+
+This section gives you a complete view at the properties that are maintained as part of widget. When interacting with a widget (on get, on create, etc) you will find most of these properties available to you, although they may be organized in a slightly different structure (ie: on get, everything but the `id` is nested inside an `attributes` object).
+
+You can find more details in the [source code](https://github.com/resource-watch/widget/blob/master/app/src/models/widget.model.js).
+
+Field name              | Type           | Required             | Default value              | Description                                                                  
+----------------------- | -------------- | -------------------- |----------------------------| ---------------------------------------------------------------------------- 
+id                      | String         | Yes (autogenerated)  |                            | Unique Id of the widget. Auto generated on creation. Cannot be modified by users.
+dataset                 | String         | Yes                  |                            | Id of the dataset to which the widget corresponds. Set on widget creation, cannot be modified.   
+name                    | String         | Yes                  |                            | Name of the widget.                                                         
+slug                    | String         | Yes (autogenerated)  |                            | Slug of the widget. Auto generated on creation. Cannot be modified by users.        
+userId                  | String         | Yes (autopopulated)  |                            | Id of the user who created the widget. Set automatically on creation. Cannot be modified by users.
+description             | String         | No                   |                            | User defined description of the widget.
+source                  | String         | No                   |                            | Description of the source of the widget's data, as it's meant to be displayed to the end user.
+sourceUrl               | String         | No                   |                            | URL of the source of the data, as it's meant to be displayed to the end user.
+authors                 | String         | No                   |                            | Author or authors of the data displayed on the widget, as it's meant to be displayed to the end user.
+queryUrl                | String         | No                   |                            | URL of the RW API query or external URL containing the data displayed on the widget
+thumbnailUrl            | String         | No                   |                            | URL of a example thumbnail of the rendered widget.
```

Comment:
Can the `thumbnailUrl` property be set by end-users?

Reply:
technically it can be updated by the user, but the update process triggers a thumbnail autogeneration, so it won't hold the user defined value for long